pages/contribute/editors_checklist.md

```diff
@@ -11,11 +11,10 @@ summary: Checklist for editors before approving and merging a pull request ({% g
    * unique `page_id` ([List of page IDs](website_overview))
    * `contributors`
    * `related_pages` ([Related pages](editorial_board_guide.html#related-pages))
-   * `training`
    * `search_exclude` must be deleted
    * `description`
    * `affiliation`
```

Comment:
```suggestion
```

Reply:
I believe this is linked with https://github.com/elixir-europe-training/ELIXIR-Training-SPLASH/pull/132 for which this field is required, it'd propose keeping it.